### PR TITLE
docs: minor wording & formatting tweaks

### PR DESCRIPTION
**Pull request by morehawes**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)

### 📚 Description

Minor wording changes and formatting fixes for Guide & API sections.

### 📝 Checklist

- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/3.api/2.composables/use-fetch.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ const { data, pending, error, refresh } = await useFetch('https://api.nuxtjs.dev
 })
 ```
 
-Results in `https://api.nuxtjs.dev/mountains?param1=value1&param2=value2`
+Results in `https://api.nuxtjs.dev/mountains?param1=value1&param2=value2`.
```
Comment (DamianGlowala) [inline docs/3.api/2.composables/use-fetch.md:39]: Might sound a little bit nicer.
```suggestion
The above example results in `https://api.nuxtjs.dev/mountains?param1=value1&param2=value2`.
```

File: docs/2.guide/3.going-further/3.modules.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ export default defineNuxtModule((options, nuxt) => {
 
 However, **we do not recommend** using this low-level function definition. Instead, to define a module, **we recommend** using the object-syntax with `meta` property to identify your module, especially when publishing to npm.
 
-This helper makes writing Nuxt Module more straightforward by implementing many common patterns seen in modules, guaranteeing future compatibility, and improving your module author developer experience and the one of your module users.
+This helper makes writing Nuxt Module more straightforward by implementing many common patterns seen in modules, guaranteeing future compatibility, improving the experience for both module authors and users.
```
Comment (danielroe) [inline docs/2.guide/3.going-further/3.modules.md:140]:
```suggestion
This helper makes writing Nuxt modules more straightforward by implementing many common patterns needed by modules, guaranteeing future compatibility and improving the experience for both module authors and users.
```

File: docs/3.api/1.components/1.nuxt-client-fallback.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "<NuxtClientFallback>"
-description: "Nuxt provides `<NuxtClientFallback>` component to render its content on the client if any of its children trigger an error in SSR"
+description: "Nuxt provides <NuxtClientFallback> component to render its content on the client if any of its children trigger an error in SSR"
```
Comment (danielroe) [inline docs/3.api/1.components/1.nuxt-client-fallback.md:3]:
```suggestion
description: "Nuxt provides the <NuxtClientFallback> component to render its content on the client if any of its children trigger an error in SSR"
```

File: docs/3.api/1.components/8.nuxt-island.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "<NuxtIsland>"
-description: "Nuxt provides `<NuxtIsland>` component to render a non-interactive component without any client JS"
+description: "Nuxt provides <NuxtIsland> component to render a non-interactive component without any client JS."
```
Comment (danielroe) [inline docs/3.api/1.components/8.nuxt-island.md:3]:
```suggestion
description: "Nuxt provides the <NuxtIsland> component to render a non-interactive component without any client JS."
```

**Review by danielroe — APPROVED** [review]
Thank you!

**Pull request merged**